### ISSUE TITLE
require boto 2.6.0

### ISSUE THREAD

**Issue by davidmarin** [Yelp/mrjob#1079]
In theory, mrjob 0.4.x supports boto 2.2.0, but if you actually call `ListAttachedRolePolicies`, you get this error:

```xml
boto.exception.BotoServerError: BotoServerError: 400 Bad Request
<ErrorResponse xmlns="https://iam.amazonaws.com/doc/2010-05-08/">
  <Error>
    <Type>Sender</Type>
    <Code>InvalidAction</Code>
    <Message>Operation ListAttachedRolePolicies is not enabled for signature version 2</Message>
  </Error>
  <RequestId>4f481af4-273a-11e5-888b-7784cc961537</RequestId>
</ErrorResponse>
```

This was fixed in boto 2.6.0.

boto 2.2.0 is so old you can't even `pip install` it anymore, and using old security methods is a liability, so just going to require people to upgrade, at least a little.

**Comment by davidmarin** [comment]
This was fixed in boto 2.6.0.

Let's just require `boto>=2.6.0`. I don't think it'll be a practical problem for anyone.

**Comment by davidmarin** [comment]
Have this ready in my `boto-2.6.0` branch. Holding off on merging for now, don't want to mess up pull requests.

**Comment by davidmarin** [comment]
Merged `boto-2.6.0`
